lib/response.js

```diff
@@ -22,7 +22,7 @@ module.exports = (function () {
             this.handler.response = buildSpeechletResponse({
                 sessionAttributes: this.attributes,
                 output: getSSMLResponse(speechOutput),
-                reprompt: getSSMLResponse(repromptSpeech),
+                reprompt: getSSMLResponse(repromptSpeech || speechOutput),
```

Comment:
There may be a case where the user doesn't want to specify a reprompt, it's marked optional in the [Alexa Skills kit documentation](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/alexa-skills-kit-interface-reference) and in that case the user isn't reprompted